From f1edd60447ee88a1c8584f47782b0e33c809f856 Mon Sep 17 00:00:00 2001
From: Tommy Webb <tommy@calyxinstitute.org>
Date: Wed, 8 Mar 2023 14:37:56 -0500
Subject: [PATCH 11/15] Disable Privacy Sandbox feature

Always return that Privacy Sandbox is restricted. This disables the
feature and removes it from the settings UI.

Change-Id: Ia74053b190d14e5e98db68f0cea81a32673e0453
---
 .../browser/privacy_sandbox/privacy_sandbox_settings_delegate.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/privacy_sandbox/privacy_sandbox_settings_delegate.cc b/chrome/browser/privacy_sandbox/privacy_sandbox_settings_delegate.cc
index 77d46136af268..600d954c3cbeb 100644
--- a/chrome/browser/privacy_sandbox/privacy_sandbox_settings_delegate.cc
+++ b/chrome/browser/privacy_sandbox/privacy_sandbox_settings_delegate.cc
@@ -69,6 +69,7 @@ bool PrivacySandboxSettingsDelegate::IsRestrictedNoticeEnabled() const {
 }
 
 bool PrivacySandboxSettingsDelegate::IsPrivacySandboxRestricted() const {
+  if ((true)) return true;
   // If the Sandbox was ever reported as restricted, it is always restricted.
   // TODO (crbug.com/40262264): Adjust when we have a graduation flow.
   bool was_ever_reported_as_restricted =
-- 
2.47.3

